cmd/create/cluster/cmd.go

```diff
@@ -764,105 +766,71 @@ func run(cmd *cobra.Command, _ []string) {
 	hasRoles := false
 	if isSTS && roleARN == "" {
 		minor := ocm.GetVersionMinor(version)
-		role := aws.AccountRoles[aws.InstallerAccountRole]
 
+		var spin *spinner.Spinner
+		if r.Reporter.IsTerminal() {
+			spin = spinner.New(spinner.CharSets[9], 100*time.Millisecond)
+		}
+		if spin != nil {
+			if !output.HasFlag() && r.Reporter.IsTerminal() {
+				r.Reporter.Infof("Fetching acc roles for the aws account: %s", awsCreator.AccountID)
```

Comment:
Since this is going to be displayed to the users:
```suggestion
				r.Reporter.Infof("Fetching account roles for the aws account: %s", awsCreator.AccountID)
```